src/stdlib/range.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Range
+---
+
+Utilities for working with ranges.
+
+```grain
+import Range from "range"
+```
+
+## Types
+
+Type declarations included in the Range module.
+
+### Range.**Range**
+
+```grain
+enum Range {
+  Inclusive(Number, Number),
+  Exclusive(Number, Number),
+}
+```
+
+Ranges can be inclusive or exclusive.
+
+## Values
+
+### Range.**inRange**
+
+```grain
+inRange : (Number, Range) -> Bool
+```
+
+Checks if the given number is within the range.
+
+### Range.**forEach**
+
+```grain
+forEach : ((Number -> Void), Range) -> Void
+```
+
+Calls the given function with each number in the range.
```

Comment:
```suggestion
Calls the given function with each number in the range. Always increases the value by `1` in each iteration, even if non-integer values were provided in the range.
```

Reply:
It also decreases by 1 if the range is backwards. I'll add that in:
```suggestion
Calls the given function with each number in the range. For increasing ranges, the value is increased by `1` in each iteration, and for decreasing ranges, the value is decreased by `1`. The value is always changed by `1`, even if non-integer values were provided in the range.
```